src/Components/Data_Array.js

```diff
@@ -0,0 +1,122 @@
+    title: 'Grand Theft Auto V',
+    tags: [
+      { tag: 'sandbox', id: 1 },
+      { tag: 'adventure', id: 2 },
+      { tag: 'TPS', id: 3 },
+    ],
+  },
+  {
+    src:
+      'https://www.fifplay.com/img/public/fifa-23-cover-ultimate-edition.jpg',
+    title: 'FIFA 23',
+    tags: [
+      { tag: 'football', id: 1 },
+      { tag: 'sports', id: 2 },
+      { tag: 'simulation', id: 3 },
+    ],
+  },
+  {
+    src:
+      'https://www.mobygames.com/images/covers/l/601919-call-of-duty-modern-warfare-playstation-4-front-cover.png',
+    title: 'Call of Duty 4: Modern Warfare',
+    tags: [
+      { tag: 'survival', id: 1 },
+      { tag: 'tactical shooter', id: 2 },
+      { tag: 'FPS', id: 3 },
+    ],
+  },
+  {
+    src:
+      'https://www.mobygames.com/images/covers/l/497224-super-mario-64-wii-u-front-cover.jpg',
+    title: 'Super Mario 64',
+    tags: [
+      { tag: 'platform', id: 1 },
+      { tag: 'adventure', id: 2 },
+      { tag: 'fighting', id: 3 },
+    ],
+  },
+  {
+    src:
+      'https://sonicthehedgehog.com/wp-content/uploads/2022/05/Sonic_DashPlus.jpg',
+    title: 'Sonic Dash +',
+    tags: [
+      { tag: 'maze', id: 1 },
+      { tag: 'action', id: 2 },
+      { tag: 'multiplayer', id: 3 },
+    ],
+  },
+  {
+    src: 'https://play-lh.googleusercontent.com/JRd05pyBH41qjgsJuWduRJpDeZG0Hnb0yjf2nWqO7VaGKL10-G5UIygxED-WNOc3pg',
+    title: 'PUBG',
+    tags: [
+      { tag: 'battle royale', id: 1 },
+      { tag: 'action-adventure', id: 2 },
+      { tag: 'shooter', id: 3 },
+    ],
+  },
+  {
+    src:
+      'https://www.mobygames.com/images/covers/l/487051-fortnite-battle-royale-nintendo-switch-front-cover.jpg',
+    title: 'Fortnite',
+    tags: [
+      { tag: 'battle royale', id: 1 },
+      { tag: 'survival', id: 2 },
+      { tag: 'action', id: 3 },
+    ],
+  },
+  {
+    src:
+      'https://image.api.playstation.com/cdn/UP1004/CUSA03041_00/Hpl5MtwQgOVF9vJqlfui6SDB5Jl4oBSq.png',
+    title: 'Red Dead Redemption 2',
+    tags: [
+      { tag: 'western', id: 1 },
+      { tag: 'fiction', id: 2 },
+      { tag: 'action-adventure', id: 3 },
+    ],
+  },
+  {
+    src:
+      'https://images.rbxcdn.com/cece570e37aa8f95a450ab0484a18d91',
+    title: 'Roblox',
+    tags: [
+      { tag: 'sandbox', id: 1 },
+      { tag: 'action-adventure', id: 2 },
+      { tag: 'open world', id: 3 },
+    ],
+  },
+  {
+    src:
+      'https://image.api.playstation.com/vulcan/ap/rnd/202211/1512/fcQBW9igeauBKr0QZUSPXmyX.png',
+    title: 'Fall Guys',
+    tags: [
+      { tag: 'battle royale', id: 1 },
+      { tag: 'simulation', id: 2 },
+      { tag: 'platform', id: 3 },
+    ],
+  },
+  {
+    src:
+      'https://image.api.playstation.com/vulcan/ap/rnd/202211/1900/WwXQmYTM3fvSRSvVzVbrnmdD.png',
+    title: 'Rocket League',
+    tags: [
+      { tag: 'simulation', id: 1 },
+      { tag: 'action', id: 2 },
+      { tag: 'racing', id: 3 },
+    ],
+  },
+]
+export default array
```

Comment:
Rename this variable and change file name to GameMockData
```suggestion
export default gameList
```